### DISCUSSION TITLE
The f-droid release isn't getting updated

### DISCUSSION THREAD

**Discussion by Loup-Garou911XD** [Acode-Foundation/Acode#649] [Q&A]
Its still on v1.6.0

**Comment by deadlyjack** [comment]
I don't know how to update there. :'(

**Reply by Loup-Garou911XD** [reply]
Lol,i will try to find how to
I heard they automatically build the apk from source

**Reply by deadlyjack** [reply]
Yes, but you need to update the repository and I have no Idea how to.

**Reply by Loup-Garou911XD** [reply]
One question,why don't you make releases here?

**Reply by deadlyjack** [reply]
It's too much work for me. I constantly upload updated version on play store as well as telegram. You can download latest version from telegram.
